src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Don't forget to comment on the interface. A description is useful to let the user know about the interface.